gcc/rust/util/rust-attributes.cc

```diff
@@ -37,6 +37,18 @@ Attributes::is_known (const std::string &attribute_path)
 
   return !lookup.is_error ();
 }
+bool
+Attributes::is_rustc_builtin_macro (AST::AttrVec outer_attrs)
```

Comment:
You should be using const AST::AttrVec& outer_attrs since this can be constified